addons/addon-base-raas-ui/packages/base-raas-ui/src/models/users/UsersStore.js

```diff
@@ -33,7 +33,12 @@ const UsersStore = BaseStore.named('UsersStore')
 
     return {
       async doLoad() {
-        const users = (await getUsers()) || [];
+        let users = [];
+        try {
+          users = await getUsers();
+        } catch (e) {
+          console.error('Could not get users', e);
```

Comment:
So when a guest is logged in, this will log an error to the logs but it is not an error. 

Plus, if a non guest logs in and for whatever reason the `getUsers()` call fails, it will not bubble to the UI, which is not the right behavior for a valid error. Is there any way to add these conditions?

Reply:
This code replicates the intended behavior of the original code by setting the users list to `[]` if `getUsers` fails. 

For non guest users, they should be able to call the user list API successfully and get the list of users. In the case that they can't, they're not going to be able to do anything with the error shown in the UI, so that's why I think logging the error to the console is preferable. That's what I saw was being done with the [StudiesStore](https://github.com/awslabs/service-workbench-on-aws/blob/9e224ff1aee34ccbdea14f330c7c840a9a39b125/addons/addon-base-raas-ui/packages/base-raas-ui/src/models/studies/StudiesStore.js#L135). 

To resolve the Guest use case, maybe I can specify in the `error` message that if they're a Guest, it is expected that they won't be able to call the list users API?